editor/editor_settings.cpp

```diff
@@ -459,10 +459,18 @@ void EditorSettings::_load_defaults(Ref<ConfigFile> p_extra_config) {
 	EDITOR_SETTING(Variant::FLOAT, PROPERTY_HINT_RANGE, "interface/inspector/float_drag_speed", 5.0, "0.1,100,0.01")
 
 	// Theme
-	EDITOR_SETTING(Variant::STRING, PROPERTY_HINT_ENUM, "interface/theme/preset", "Default", "Default,Breeze Dark,Godot 2,Gray,Light,Solarized (Dark),Solarized (Light),Black (OLED),Custom")
+
+	// Please keep these sorted alphabetically.
+	String dark_presets = "Black (OLED),Breeze Dark,Godot 2,Gray,Solarized (Dark)";
+	String light_presets = "Solarized (Light)";
+
+	EDITOR_SETTING(Variant::INT, PROPERTY_HINT_ENUM, "interface/theme/theme_mode", 0, "Dark,Light,Auto")
```

Comment:
```suggestion
	EDITOR_SETTING(Variant::INT, PROPERTY_HINT_ENUM, "interface/theme/theme_mode", 0, "Dark,Light,Automatic (Follow System Theme)")
```

Reply:
We have plenty of space to display option values, so I don't think it's too much of an issue. We could name the option just `Follow System Theme` too.

Reply:
I don't mean to be difficult about such a minor thing, but I think consistency in the options is important as we already have "Dark, Light, Auto" as options for the icon and font color setting. I'd certainly meet in the middle and say "System" could replace "Auto" if you think this option needs more clarity. Here's some screenshots:

![Screenshot from 2024-01-03 21-08-47](https://github.com/godotengine/godot/assets/37851452/eba9f843-40c3-4ef1-a91a-52b1033991f6)
![Screenshot from 2024-01-03 21-09-52](https://github.com/godotengine/godot/assets/37851452/e9521097-91ba-4108-ad40-b2c9d84f189c)